charts/service/values.yaml

```diff
@@ -11,7 +11,7 @@ image:
   # -- Overrides the image tag whose default is the chart appVersion.
 
 imagePullSecrets: []
-nameOverride: "{{ .Release.Name }}"
```

Comment:
why was this changed? all older ingresses would lose their name and break

Reply:
@13sh It's not applicable to set expression in values.yaml.
"Caught by Github checks"

Reply:
Does this mean the values currently used will change or not?

Reply:
Nothing will change, we override it by  helmvalues in gitops